modules/context/repo.go

```diff
@@ -103,6 +103,14 @@ func (r *Repository) CanUseTimetracker(issue *models.Issue, user *models.User) b
 		r.IsWriter() || issue.IsPoster(user.ID) || issue.AssigneeID == user.ID)
 }
 
+// CanCreateIssueDependencies returns whether or not a user can create dependencies.
+func (r *Repository) CanCreateIssueDependencies(user *models.User) bool {
+	// Checks for following:
+	// 1. Are dependencies enabled
+	// 2. Has the user write access?
```

Comment:
*does the user have write access?

Reply:
Yes? Sorry, I'm not sure if I understood you correctly. Do you mean why the user needs write access?

Reply:
It's just grammar nitpicking. I'm saying the comment is incorrect. Sorry about the nit :P

Reply:
Ahh ok, fixed it. (I just oriented on other comments)